src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface Figure {
+    public Colors getColor();
```

Comment:
color is a state, so better create abstract class Figure with field `color` instead of this interface